### PR TITLE
🏗♻️🚮 Fully remove devdash bundling

### PR DESCRIPTION
**Pull request by alanorozco**
nodemodulesirl

Not using any JSX at the moment so we can just remove.

### CODE REVIEW

File: build-system/app-index/bundler.js [hunk 1]
```diff
@@ -15,45 +15,51 @@
...
 module.exports = {
-  bundleComponent: async componentEntryFile => {
-    inputOptions.input = componentEntryFile;
-    const bundle = await rollup.rollup(inputOptions);
-    const {code} = await bundle.generate(outputOptions);
-    log('Generated bundle for',
-        colors.cyan(path.relative(rootDir, componentEntryFile)));
-    return code;
+  bundleComponent: async unusedComponentEntryFile => {
+    return ''; // for now.
```
Comment (rsimha) [inline build-system/app-index/bundler.js]: My 2¢: I think it's cleaner to delete this file and restore it when necessary. OTOH, if you're planning on restoring this code soon, maybe this cleanup is unnecessary?

File: package.json [hunk 2]
```diff
@@ -163,10 +162,6 @@
     "request-promise": "4.2.4",
     "rimraf": "2.6.3",
     "rocambole": "0.7.0",
-    "rollup": "1.3.0",
-    "rollup-plugin-babel": "4.3.2",
```
[the remark this answers is not in the document]
Comment (alanorozco) [reply package.json]: I don't think that's a problem, that directory has its own `package.json`.

https://github.com/ampproject/amphtml/blob/be351667632b7900d2aeb763dca78b19741d390e/extensions/amp-access/0.1/iframe-api/package.json#L20

[the remark this answers is not in the document]
Comment (rsimha) [reply package.json]: Gotcha. I retract my comment.

File: build-system/app-index/index.js [hunk 3]
```diff
@@ -34,31 +33,13 @@ const pc = process;
...
-  if (shouldCache && mainBundleCache) {
-    return mainBundleCache;
-  }
-  const bundle = await bundler.bundleComponent(mainComponent);
-  if (shouldCache) {
-    mainBundleCache = bundle;
-  }
-  return bundle;
+function setCacheStatus(unusedCacheStatus) {
+  // NOOP.
```
Comment (rsimha) [inline build-system/app-index/index.js]: This means you can remove `enable_dev_dashboard_cache` and all its associated logic, correct?

File: build-system/server.js [hunk 4]
```diff
@@ -19,7 +19,7 @@
  * @fileoverview Creates an http server to handle static
  * files and list directories for use with the gulp live server
  */
-const app = require(require.resolve('./app.js'));
+const app = require.resolve('./app.js');
```
Comment (rsimha) [inline build-system/server.js]: I think this can be written as 

```
const {app} = require('./app');
```

File: build-system/tasks/serve.js [hunk 5]
```diff
@@ -96,8 +94,6 @@ gulp.task(
         'cache': '  Make local resources cacheable by the browser ' +
             '(default: false)',
         'inspect': '  Run nodemon in `inspect` mode',
-        'enable_dev_dashboard_cache':
-            '  Enables the dev dashboard cache (default: false)',
```
Comment (rsimha) [inline build-system/tasks/serve.js]: Remove this flag from `gulpfile.js` too

https://github.com/ampproject/amphtml/blob/1c54451b82cb9048b566d33ea3ad99e3a820183f/gulpfile.js#L1699-L1700

File: build-system/server.js (hunk 4 above)
Comment (alanorozco) [reply build-system/server.js]: Whoops. My usage of `resolve` was vey wrong as well.

Comment (rsimha) [reply build-system/server.js]: I think this is still wrong. Note the curly braces around `{app}`.

Comment (alanorozco) [reply build-system/server.js]: That wasn't it, but culprit has been identified.

**Pull request merged**